components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java

```diff
@@ -2049,4 +2081,13 @@ public String getVendorFromExtension(String swaggerContent) {
     public String getType() {
         return null;
     }
+
+    @Override
+    public Object getRootLevelSwaggerExtension(String swaggerContent, String extensionKey) {
```

Comment:
Removed `swagger` as this method goes into AsyncAPI parser as well
```suggestion
    public Object getRootLevelExtension(String swaggerContent, String extensionKey) {
```